fedmsg_meta_fedora_infrastructure/koschei.py

```diff
@@ -63,6 +77,10 @@ def link(self, msg, **config):
             if 'collection' in msg['msg']:
                 url += '?collection=' + msg['msg']['collection']
             return url
+        elif 'koschei.collection.state.change' in msg['topic']:
```

Comment:
I thought that the topic is self-descriptive and it matches generic pattern `app.something.state.change` - it is sent when state of collection in [Koschei](https://fedoraproject.org/wiki/Koschei) changes. But sure, I'll add comment describing format and purpose of the message.